internal/domain/temporal/types/billing.go

```diff
@@ -0,0 +1,27 @@
+package models
+
+import "time"
+
+// BillingWorkflowInput represents the input for a billing workflow.
+type BillingWorkflowInput struct {
+	// CustomerID is the identifier for the customer in our system.
+	CustomerID string `json:"customer_id"`
+
+	// SubscriptionID is the identifier for the subscription in our system.
+	SubscriptionID string `json:"subscription_id"`
+
```

Comment:
remove entire temporal domain and fit in stuff in internal/temporal/dtos most likely for now